rundeckapp/src/main/groovy/com/dtolabs/rundeck/app/api/ApiRunAdhocRequest.groovy

```diff
@@ -37,10 +37,12 @@ class ApiRunAdhocRequest implements Validateable{
     String fileExtension
     Boolean nodeKeepgoing
     Integer nodeThreadcount
+    Boolean preserveQuotesOnArgumentsCheck
```

Comment:
suggest renaming this to `preserveQuotes`

Reply:
also, as this now affects the API, the `apiRunCommand` in ScheduledExecutionController should support this parameter only in newer API version.

Reply:
I think also, given newer API version, the run command API should use preserveQuotes=true by default if it is not specified, otherwise set it to false.

Reply:
@carlosrfranco yes, it should be toggled off by default.